.github/workflows/benches.yml

```diff
@@ -97,7 +97,7 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           name: "criterion"
-          path: "./target/criterion.zip"
+          path: "./target/criterion"
```

Comment:
Should the previous step that actually creates this zip file be removed as well?

Reply:
Ah, doh, I actually had that thought and still forgot 🤦 Thanks!

Reply:
Removed in https://github.com/timberio/vector/pull/7113/commits/badafb2743573dc994e8896883be09dc09ac4443